src/coreComponents/codingUtilities/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+#====================================================
```

Comment:
@paloma-martinez If you use decorate your headers with these line separators, can you use a consistent length of, let's say, 100 characters?

Reply:
Ok, I'll include it in the next modifications.